docs/commands/calculate-random-tile.asciidoc

```diff
@@ -0,0 +1,37 @@
+== calculate-random-tile 
+
+=== Description
+
++calculate-random-tile+ calculates a collection of bounding boxes that contain roughly equal distributions of node data from the specified input(s) and writes the output to a file.
+
+* +input+ - one or more OSM data inputs; specify multiple inputs by separating them with a semicolon
+* +output+ - output file to write the tile boundary to; valid formats are GeoJSON (*.geojson)
+* +random-seed+ - seeds the random number generator for consistent tile boundary selection; use -1 for no seeding; defaults to -1
+* +maxNodeCountPerTile+ - The maximum number of nodes in any tile.  The default is 1000; optional only if pixelSize is not specified
+* +pixelSize+ - The size of the pixels used in the calculation.  The default is 0.001; optional only if maxNodeCountPerTile is not specified
```

Comment:
That's a good question.  Let me look at the code a bit later and figure that out.